### DISCUSSION TITLE
Need different units setting for height and distance

### DISCUSSION THREAD

**Discussion by Lorie-Capella** [breezy-weather/breezy-weather#1000] [Ideas]
### Describe your idea

When i setuped units as nautical meters as perferred unit, cloud ceiling was also in nmi values. but i think it would be better if i can use ft as a height unit. i think people prefer nmi for distance unit would also prefer ft for height and altitude
![Screenshot_20240422-201848](https://github.com/breezy-weather/breezy-weather/assets/57549294/7882bc52-64fe-4c7a-8cee-7e44441a3d09)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.1 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
It's the same thing in metric. We never use kilometers for height, only meters. However, ceiling is a very specific data that only exists as current observations and only with one source (AccuWeather).

I was even thinking of completely removing it as it's not a very useful info if you're not in aeronautic. Is it your case?

The options I have in mind is either keep it as it is now with one unit (to not create a specific ceiling unit for such an edge case), or just remove it.
